### PR TITLE
add data partition codes

### PR DESCRIPTION
**Pull request by vincent17890**
I've added data partition codes that divide the training data into training set, validation set, and test set.

### CODE REVIEW

**Comment by vincent17890** [comment]
I've added codes that drop nan from the data set and run three regression algorithms on the dataset.

**Review by lawrence-j-liu — APPROVED** [review]
